cpp/lib/src/util.cc

```diff
@@ -44,11 +66,14 @@ char *progname; // Must be set in main() with "progname = argv[0];";
 const std::string Logger::FUNCTION_NAME_SEPARATOR(" --> ");
 
 
-
 Logger::Logger()
     : fd_(STDERR_FILENO), log_process_pids_(false), log_no_decorations_(false), log_strip_call_site_(false),
       min_log_level_(LL_INFO)
 {
+    log_fd = fd_;
```

Comment:
How about calling redirectOutput (fd_) instead?

Reply:
Obsolete because fd_ does not exist any longer.